src/app/features/profile/profile-list/profile-list.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import {  getUserProfile } from '@store/selectors';
+import { Store } from '@ngrx/store';
+import { AppState } from '@store/reducers';
+import { loadProfile } from '@store/actions';
+import { UserProfile } from '@interfaces';
+
+@Component({
+    selector: 'app-profile-list',
+    styleUrls: ['./profile-list.component.less'],
+    templateUrl: './profile-list.component.html'
+})
+export class ProfileListComponent implements OnInit {
+
+    users: UserProfile [] = [];
+
+    constructor (private store: Store<AppState>) {}
+
+    ngOnInit () {
+
+        for (let i = 0 ; i < 10; i++) {
+
+            this.store.dispatch(loadProfile());
+
+        }
+        this.store.select(getUserProfile).subscribe((data) => {
```

Comment:
user profile is a name for a single user profile
but here you are using it as name for a multiple users state